cli/command/volume/inspect_test.go

```diff
@@ -0,0 +1,150 @@
+package volume
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/cli/internal/test"
+	// Import builders to get the builder function as package function
+	. "github.com/docker/docker/cli/internal/test/builders"
+	"github.com/docker/docker/pkg/testutil/assert"
+	"github.com/docker/docker/pkg/testutil/golden"
+)
+
+func TestVolumeInspectErrors(t *testing.T) {
+	testCases := []struct {
+		args              []string
+		flags             map[string]string
+		volumeInspectFunc func(volumeID string) (types.Volume, error)
+		expectedError     string
+	}{
+		{
```

Comment:
I think initialising this table value with nil args/volumeInspectFunc would be easier to read over time.